### PR TITLE
Hello Microverse Project

### PR DESCRIPTION
**Pull request by Godfrey-Alimony**
Add markup for the Homepage.
Add stying css file for the Homepage.
Add screenshot for the website.
Add README File for the website.
Add MIT File for the website.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,78 @@
...
+
+`./src/assets` all asests and images files
+
+`./src/css` global css, including fonts.css
+
+`./src/js` global javascript files
+
+`gulpfile.js` the build script
+
+`package.json` dependency of the modules
```
[the remark this answers is not in the document]
Comment (zieeco) [reply README.md:52]: **[REQUIRED CHANGES]**

- Please, kindly rename the title of your PR to be more descriptive. e.g `Hello Microverse`, `Hello Microverse Project`, `Greetings`, etc

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Godfrey-Alimony 


Great work with the hello microverse project 👏 
You've done well using the correct gitflow, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Good commits message. ✔️



## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Godfrey-Alimony) [reply README.md:52]: Done sir.

File: README.md [hunk 2]
```diff
@@ -1 +1,78 @@
-# Hello-World
\ No newline at end of file
+[@githubhandle](https://github.com/Godfrey-Alimony)
+
+Project Name: Hello-World Webpage
```
[the remark this answers is not in the document]
Comment (Godfrey-Alimony) [reply README.md:3]: Done sir

**Comment by Godfrey-Alimony** [comment]
I have done the correction sir and thank for you greatful advice.

**Comment by Godfrey-Alimony** [comment]
I have commited all the changes you told me to correct sir.Thank

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+node_modules/
```
[the remark this answers is not in the document]
Comment (sja-thedude) [reply .gitignore:1]: Implemented 👍

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Godfrey-Alimony, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Good job so far

 ## Required Changes ♻️
- [x] Kindly add a descriptive Title as suggested by the previous reviewer. Replace `My first commit` with `Hello Microverse Project`.
2022-05-24 (2)_LI


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Godfrey-Alimony** [comment]
Hello, I have corrected the PR title. Thank

**Comment by arslanbisharat** [comment]
Hi,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Godfrey-Alimony** [comment]
You most welcome Thank

**Pull request merged**